src/event_loop.rs

```diff
@@ -193,7 +193,7 @@ impl ControlFlow {
 impl Default for ControlFlow {
     #[inline(always)]
     fn default() -> Self {
-        Self::Poll
+        Self::Wait
```

Comment:
Could you annotate the default on the enum instead of implementing it?

Reply:
Done.